### ISSUE TITLE
RStudio terminal and PuTTY terminal return different values for R_LIBS_USER

### ISSUE THREAD

**Issue by hzarkoob** [rstudio/rstudio#3655]
When running the (simple) command `Rscript -e 'print(Sys.getenv("R_LIBS_USER"))'` I get different results in the PuTTY terminal vs the Rstudio built-in terminal.

The output on the PuTTY terminal:
`"~/R/x86_64-pc-linux-gnu-library/3.5"`

The output on the RStudio terminal:
`"~/R/x86_64-pc-linux-gnu-library/3.4"`

As demonstrated, the output in the RStudio terminal refers to an older version of R. 

PS. I checked all the files with the phrase "Renviron" in their name or their extension. Interestingly, I was not able to file ANY file that sets the R_LIBS_USER variable to the `"~/R/x86_64-pc-linux-gnu-library/3.4"`, as least in this explicit form.

### System details

    RStudio Edition : Server
    RStudio Version : 1.1.423
    OS Version      : Ubuntu 16.04.5
    R Version       : 3.5.1

========================================
PS. Thank you for the great RStudio product :)

**Comment by jmcphers** [comment]
What result do you get in the R console in RStudio? (i.e. just running `Sys.getenv("R_LIBS_USER")`)?

Does this string appear anywhere in your `/etc/rstudio` directory?

**Comment by hzarkoob** [comment]
On the R console, I get `"~/R/x86_64-pc-linux-gnu-library/3.5"`

- The above string appears neither in `etc/rstudio/rsession.conf` nor `etc/rstudio/rserver.conf`.

**Comment by jmcphers** [comment]
Interesting! Are you using the same shell in PuTTY and the RStudio terminal?

Does that string appear anywhere in your system or user level shell profile scripts?

**Comment by hzarkoob** [comment]
I am quite new to Linux and I am not too sure, but I think the shells are the same. They show the same username and ip address. Also, the prompt schemes I saved in .bashrc are reflected in both PuTTY and RStudio terminals.

I searched for the string `~/R/x86_64-pc-linux-gnu-library/3.4` throughout the entire system using the `grep` command. Three suspicious places where I found the string are:

- /mnt/projectDirectory/.Rproj.user/6E7C8C77/console06/856423C0:[1] "~/R/x86_64-pc-linux-gnu-library/3.4"
- /mnt/projectDirectory/.Rproj.user/6E7C8C77/console06/856423C0.env:    "R_LIBS_USER" : "~/R/x86_64-pc-linux-gnu-library/3.4",
- /mnt/projectDirectory/.Rproj.user/6E7C8C77/suspended-session-data/environment_vars:R_LIBS_USER="~/R/x86_64-pc-linux-gnu-library/3.4"

**Comment by kevinushey** [comment]
The presence of `R_LIBS_USER` in that `environment_vars` file suggests that RStudio is saving and loading the `R_LIBS_USER` associated with a previous session (in this case, one that was run with R 3.4).

Does restarting R not clear that state? If not, you can try moving the `suspended-session-data` directory out of the way so RStudio will no longer use it.

**Comment by hzarkoob** [comment]
@kevinushey Thank you for your comment. It was helpful!

Here is a description of the steps I went through and the responses I got:
- Restarted R => It did not help (still getting 3.4 path).
- Temporarily removed the `suspended-session-data` folder. => It did not help (still getting 3.4 path).
- Removed the entire `6E7C8C77` folder and then copied it back. => The issue got solved!! That is, I get the 3.5 path when running the command `Rscript -e 'print(Sys.getenv("R_LIBS_USER"))'`. 

However, with that change, I started to get errors when trying to work with the files that were open before the above steps were taken. The error was saying something like: "No such file is found". (I do not remember the exact message).

- I overcame the above issue by fully deleting the `.Rproj.user` folder and re-creating an RStudio project. 

Everything seems to be in a good shape now.

PS. I was restarting R after taking any step.
